# The outgoing command message from coalition leader

uint16[] receiverRobotID

uint64 sendingTime #The sending time

uint16 cmdTypeID

string cmdMessage
